clarity/src/vm/ast/parser/v2/lexer/mod.rs

```diff
@@ -2093,6 +2136,7 @@ mod tests {
             }
         );
 
+        /* cylewitruk: Commenting this out as `~` isn't documented anywhere...
```

Comment:
Similarly here, replace the `~` with `@` in this test.

Reply:
Done!